x-pack/plugins/actions/server/saved_objects/migrations.test.ts

```diff
@@ -15,121 +15,185 @@ import { migrationMocks } from 'src/core/server/mocks';
 const context = migrationMocks.createContext();
 const encryptedSavedObjectsSetup = encryptedSavedObjectsMock.createSetup();
 
-describe('7.10.0', () => {
+describe('successful migrations', () => {
```

Comment:
Recommend reviewing this with `Hide whitespace changes` turned on for more meaningful diffs.